worker/transcribee_worker/worker.py

```diff
@@ -119,22 +119,24 @@ def keepalive(self, task_id: str, progress: Optional[float]):
     async def perform_task(self, task: AssignedTask):
         logging.info(f"Running task: {task=}")
 
+        def progress_callback(progress, step=task.task_type, extra_data=None):
```

Comment:
Let's enforce using kwargs for clarity and forward compatiblity

```suggestion
        def progress_callback(*, progress, step=task.task_type, extra_data=None):
```